REQUIRE

```diff
@@ -1,7 +1,6 @@
-julia 0.4
+julia 0.6
 BenchmarkTools 0.0.7
```

Comment:
Shouldn't BenchmarkTools be also updated with the new minimum after it's tagged of course

Reply:
Yes. It isn't tagged yet.